internal/controller/operator/factory/reconcile/statefulset_pvc_expand.go

```diff
@@ -222,8 +222,7 @@ func mayGrow(ctx context.Context, pvc *corev1.PersistentVolumeClaim, newSize, ex
 		// do no return error
 		// probably, user updated pvc manually
 		// without applying this changes to the configuration.
-		logger.WithContext(ctx).Info(fmt.Sprintf("cannot decrease PVC=%s size from=%s to=%s", pvc.Name, newSize.String(), existSize.String()),
-			"cannot decrease PVC size, please check VolumeClaimTemplate configuration")
+		logger.WithContext(ctx).Info("cannot decrease PVC size, please check VolumeClaimTemplate configuration", "pvc", pvc.Name, "from", existSize.String(), "to", newSize.String())
```

Comment:
It's better to use `fmt.Sprintf` instead of each field logging.

The problem with per field logging is read-ability.  Log line:
`cannot decrease PVC size, please check VolumeClaimTemplate configuration` doesn't have any context with out sizes and PVC name. It also adds an additional pressure on remote logging systems, since each field needs an additional index.

Latter we're going to replace logger implementation.

 See this issue for more context https://github.com/VictoriaMetrics/operator/issues/1191

Reply:
Good point! Updated